src/main/java/com/questhelper/panel/questorders/QuestOrders.java

```diff
@@ -48,9 +48,9 @@ public class QuestOrders
 		QuestHelperQuest.IMP_CATCHER,
 		QuestHelperQuest.WITCHS_POTION,
 		QuestHelperQuest.GERTRUDES_CAT,
+		QuestHelperQuest.CHILDREN_OF_THE_SUN,
```

Comment:
would be nice to undo all quest order changes for this pr, and have that as a separate PR instead, just for commit sanity once it lands in the main branch